package.json

```diff
@@ -38,6 +38,7 @@
     "lodash.capitalize": "4.2.1",
     "lodash.clonedeep": "4.5.0",
     "lodash.debounce": "4.0.8",
+    "lodash.isnil": "^4.0.0",
```

Comment:
Does this dependency also need to be pinned?

Reply:
Doesn't _have_ to be, but for convention, yes it should be :) Good callout.